.eslintrc.json

```diff
@@ -28,7 +28,8 @@
     "storybook/no-uninstalled-addons": [
       "error",
       {
-        "ignore": ["@nx/react/plugins/storybook"]
+        "ignore": ["@nx/react/plugins/storybook"],
+        "packageJsonLocation": "../../package.json"
```

Comment:
Need to point the Storybook lint rule to the root `package.json`, or else it'll pick up the package one (e.g. `packages/vue/package.json`) which results in missing dependencies being reported.